content/doc/advanced/customizing-gitea.en-us.md

```diff
@@ -17,14 +17,32 @@ menu:
 
 The main way to customize Gitea is by using the `custom` folder. This is the central place to override and configure features.
 
+If you installed Gitea from binary, you can find the `custom` folder next to the binary.
+You should already find it, with a `conf` folder inside, where Gitea stores all the configuration settings provided through the installation steps.
+
```

Comment:
Which binary contains the `custom` folder ? Last time I checked he official binary downloads were a single executable file...